molecule/default/molecule.yml

```diff
@@ -9,7 +9,14 @@ lint: |
     flake8
 platforms:
   - name: instance
-    image: centos:7
+    image: rockylinux:9
```

Comment:
Proposing to remove all the lines below

Reply:
fixed in https://github.com/ome/ansible-role-cli-utils/pull/9/commits/24384efaff54cd51349992a38561cd9227161ccf